mappings/com/mojang/blaze3d/systems/RenderSystem.mapping

```diff
@@ -1,166 +1,119 @@
 CLASS com/mojang/blaze3d/systems/RenderSystem
 	FIELD MAX_SUPPORTED_TEXTURE_SIZE maxSupportedTextureSize I
-	FIELD MINIMUM_ATLAS_TEXTURE_SIZE MINIMUM_ATLAS_TEXTURE_SIZE I
 	FIELD RENDER_THREAD_TESSELATOR RENDER_THREAD_TESSELLATOR Lnet/minecraft/unmapped/C_hiiunyvh;
-	FIELD apiDescription apiDescription Ljava/lang/String;
-	FIELD isInInit isInInit Z
-	FIELD lastDrawTime lastDrawTime D
 	FIELD modelViewStack MODEL_VIEW_STACK Lorg/joml/Matrix4fStack;
-	FIELD pollEventsWaitStart pollEventsWaitStart Ljava/util/concurrent/atomic/AtomicLong;
-	FIELD pollingEvents pollingEvents Ljava/util/concurrent/atomic/AtomicBoolean;
```

Comment:
can't suggest on deleted lines, but these are `static final`

Reply:
the uppercasing doesn't apply to atomics (i forgot too, thanks linter)

Reply:
oh, didn't know, whoops